mapper/tedge_mapper/src/sm_c8y_mapper/tests.rs

```diff
@@ -0,0 +1,408 @@
+#[cfg(test)]
+#[cfg(feature = "integration-test")]
+#[cfg(feature = "mosquitto-available")]
+
+mod tests {
+    use crate::component::TEdgeComponent;
+    use mqtt_client::{Client, MqttClient, Topic, TopicFilter};
+    use once_cell::sync::OnceCell;
+    use serial_test::serial;
+    use std::io::Write;
+    use std::process::{Child, Stdio};
+    use std::time::Duration;
+    use tedge_config::{ConfigRepository, TEdgeConfig, TEdgeConfigLocation};
+
+    const MQTT_TEST_PORT: u16 = 56666;
+    static SERVER: OnceCell<Child> = OnceCell::new();
+
+    #[tokio::test]
+    #[serial]
+    async fn mapper_publishes_a_software_list_request() {
+        // Make sure the server is running.
+        let _start_mosquitto = SERVER.get_or_init(spawn_mqtt_server);
```

Comment:
For the time being till we have teh solution for spawning the server automatically this should be removed.

Reply:
True, thanks for catching it.